server/database.json

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
copy the version of the database we have in main. we dont want any merge conflicts

Reply:
Alright. Thing is because both the cart page uses data from the database, once you test it, it will modify the database. I will replace the database.json though